build.sbt

```diff
@@ -2,13 +2,13 @@ organization := "zamblauskas"
 
 name := "sbt-examplestest"
 
-scalaVersion := "2.12.4"
+scalaVersion := "2.12.10"
```

Comment:
Was not able to update to Scala `2.13.1` because `scalamock-scalatest-support` has not been released for Scala 2.13. Not sure if that is still needed, because in the [scalamock README](https://github.com/paulbutcher/ScalaMock#using-scalamock) it says that it is possible to use scalamock and scalatest without an additional glue library.